js/globals.ts

```diff
@@ -71,3 +72,8 @@ window.TextEncoder = textEncoding.TextEncoder;
 export type TextEncoder = textEncoding.TextEncoder;
 window.TextDecoder = textEncoding.TextDecoder;
 export type TextDecoder = textEncoding.TextDecoder;
+
+window.workerMain = workers.workerMain;
+// TODO These shouldn't be available in main isolate.
```

Comment:
I think we will have to create a seperate global module for the compiler isolate when we get there.  There is clearly a lot of stuff that isn't necessary in order to support just the compiler.